planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/script.js

```diff
@@ -0,0 +1,112 @@
+//VERSION=3
+
+// Set defaultVis to false to scale and set color_min and color_max values.
+// LST has two observations per day: 1h30 and 13h30 solar local time
+
+const date = "2022-12-31"; // The date for which the backward average is calculated
+const nDays = 20; // The number of days to load data for
+const scaleFactor = 100; // The scale factor for the SWC values
+const defaultVis = true; // If true, the default visualization will be applied. If false, the color_min and color_max values will be used.
```

Comment:
Here it's the same as for SWC, we can now set the min/max directly in the ColorRamp, so no need to do it ourselves.

```suggestion
```

Reply:
Initially I did not add this to be able to have a `defaultVis` just like the other LST custom script examples. But I agree it would be better to immediately include the update in the `ColorRampVisualizer` initialization. Therefore, I have updated it.